### PR TITLE
[resolves #2095] Monitor AWS object build up

### LINKED ISSUE #2095: Monitor AWS object build up
Now that we are creating AWS objects per client, it is possible that stale objects accumulate and thus incur unnecessary cost. There should be a test that fails when there are more than a given number of objects for each type.

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: itests/common/src/main/java/org/wildfly/camel/test/common/aws/AWSUtils.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package org.wildfly.camel.test.common.aws;
```
Comment (jamesnetherton) [inline itests/common/src/main/java/org/wildfly/camel/test/common/aws/AWSUtils.java]: Missing license

File: itests/common/src/test/java/org/wildfly/camel/test/common/aws/AWSUtilsTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package org.wildfly.camel.test.common.aws;
```
Comment (jamesnetherton) [inline itests/common/src/test/java/org/wildfly/camel/test/common/aws/AWSUtilsTest.java]: Missing license

File: itests/common/src/main/java/org/wildfly/camel/test/common/aws/AWSUtils.java (hunk 1 above)
Comment (ppalaga) [reply itests/common/src/main/java/org/wildfly/camel/test/common/aws/AWSUtils.java]: fixed in 8614b8e

File: itests/common/src/test/java/org/wildfly/camel/test/common/aws/AWSUtilsTest.java (hunk 2 above)
Comment (ppalaga) [reply itests/common/src/test/java/org/wildfly/camel/test/common/aws/AWSUtilsTest.java]: fixed in 8614b8e

**Comment by tdiesler** [comment]
ok to test

**Comment by jamesnetherton** [comment]
retest this please

**Comment by jamesnetherton** [comment]
retest this please

**Comment by jamesnetherton** [comment]
retest this please

**Comment by jamesnetherton** [comment]
retest this please

**Comment by ppalaga** [comment]
Added 6e396de as an extra commit so that it can be eventually reverted in the future.

**Pull request closed without merging**